Druid/DRScope.class.st

```diff
@@ -2,9 +2,164 @@ Class {
 	#name : 'DRScope',
 	#superclass : 'Object',
 	#instVars : [
-		'outerScope'
+		'argumentNames',
```

Comment:
Do we need this variable? Could not be generated?

Reply:
The LoadArguments, as Stores/Load Temps, should refer to the scope where they are defined.